en/services/component_information.md

```diff
@@ -59,17 +55,21 @@ Generally this means that new versions may add fields but should not delete them
 The schema are currently a work in progress and can be modified as needed.
 Once accepted, they will be under change control (*managed* in a similar way to standard MAVLink messages).
 
+
 ## File Locations/URLs
 
-[Version metadata](#COMP_METADATA_TYPE_VERSION) *must* be stored on the device, while other component information files may be hosted on either the device or on the internet.
+[General metadata](#COMP_METADATA_TYPE_GENERAL) files and [peripherals metadata](#COMP_METADATA_TYPE_PERIPHERALS) files (if supported) *must* be stored on the device, and will usually be [**.xz** compressed](#file-compression).
+The location of these files is returned in the [COMPONENT_INFORMATION](#COMPONENT_INFORMATION) `general_metadata_uri` and `peripherals_metadata_uri` fields, respectively.
+
+Other component information files may be hosted on either the device or on the internet.
 
 > **Note** Where permitted by memory constraints you should host component information on the device (so that it is always available and cannot get out of sync).
 
-A URI string indicating the file location is returned in the [COMPONENT_INFORMATION.metadata_uri](#COMPONENT_INFORMATION) field:
-- Files on the Internet are downloaded using HTTPS or HTTP via a normal web URL (e.g. `https://some_domain/component_information/parameters.json`).
-- Files on the device are downloaded using MAVLink FTP.
-  The URI format is defined in [MAVLink FTP URL Scheme](../services/ftp.md#mavlink-ftp-url-scheme).
-  A typical parameter metadata URI might look like this: `mftp://[;comp=1]/component_information/parameters.json.xz`.
+Files on the device are downloaded using [MAVLink FTP](../services/ftp.md).
+The URI format is defined in [MAVLink FTP URL Scheme](../services/ftp.md#mavlink-ftp-url-scheme).
+A typical parameter metadata URI might look like this: `mftp://[;comp=1]/component_information/parameters.json.xz`.
+
```

Comment:
Do you also want to mention and link to the CRC (`[MAVLink CRC32](../guide/crc.md)`)?

Reply:
Good point. I created a new section for talking about caching, and that links to this.